movement/kinematics.py

```diff
@@ -362,10 +362,7 @@ def compute_forward_vector_angle(
     right_keypoint: Hashable,
     reference_vector: xr.DataArray | ArrayLike = (1, 0),
     camera_view: Literal["top_down", "bottom_up"] = "top_down",
-    in_radians: bool = False,
-    angle_rotates: Literal[
-        "ref to forward", "forward to ref"
-    ] = "ref to forward",
+    in_degrees: bool = False,
 ) -> xr.DataArray:
     r"""Compute the signed angle between a forward and reference vector.
```

Comment:
The docstring below the summary line still says 
> The returned angles are in degrees, spanning the range :math:`(-180, 180]`, unless ``in_radians`` is set to ``True``.

We can optionally use the same phrasing as elsewhere:
> Angles are returned in radians, spanning :math:`(-\pi, \pi]` according to the ``arctan2`` convention, unless ``in_degrees`` is set to ``True``.